src/tasks/start/utils/backend/bases/apm.ts

```diff
@@ -0,0 +1,69 @@
+import { hash as namehash } from 'eth-ens-namehash'
```

Comment:
We can get rid of the bases directory. These files should live at the same level as app.ts, dao.ts, etc.